pep-0432.txt

```diff
@@ -238,11 +239,15 @@ should minimise their impact on the startup overhead.
 Experience with the importlib migration suggests that the startup time is
 dominated by IO operations. However, to monitor the impact of any changes,
 a simple benchmark can be used to check how long it takes to start and then
-tear down the interpreter::
+tear down the interpreter:
+
+.. code-block:: bash
 
    python3 -m timeit -s "from subprocess import call" "call(['./python', '-Sc', 'pass'])"
 
-Current numbers on my system for Python 3.7 (as built by the Fedora project)::
+Current numbers on my system for Python 3.7 (as built by the Fedora project):
+
+.. code-block:: shell
```

Comment:
I think you meant `console` here (`shell` is an alias for `bash`)...

Reply:
Well spotted, I did! 

Please see PR https://github.com/python/peps/pull/2729 to fix it.